aioesphomeapi/core.py

```diff
@@ -92,6 +92,12 @@
 TWO_CHAR = re.compile(r".{2}")
 
 
+ESPHOME_GATT_ERRORS = {
+    -1: "Not connected",
+    -2: "Wrong address",
```

Comment:
```suggestion
```

I guess we can drop this now based on [esphome/esphome@`f8c13aa` (#3971)](https://github.com/esphome/esphome/pull/3971/commits/f8c13aa9ed7152de1ae22e88a83f8438d642b4a3)